kale/pipeline/multi_domain_adapter.py

```diff
@@ -0,0 +1,313 @@
+# =============================================================================
+# Author: Shuo Zhou, szhou20@sheffield.ac.uk/sz144@outlook.com
+#         Haiping Lu, h.lu@sheffield.ac.uk or hplu@ieee.org
```

Comment:
You do not need to add me always as an author. MPCA is different because I wrote the Matlab code and contributed a lot to coding. Here I am just reviewing.